dev/ParallaxView/ScrollInputHelper.cpp

```diff
@@ -195,7 +195,7 @@ winrt::RichEditBox ScrollInputHelper::GetRichEditBoxParent(const winrt::Dependen
 void ScrollInputHelper::GetChildScrollerOrScrollViewer(
     const winrt::DependencyObject& rootElement,
     _Out_ winrt::Scroller* scroller,
-    _Out_ winrt::ScrollViewer* scrollViewer)
+    _Out_ winrt::Windows::UI::Xaml::Controls::ScrollViewer* scrollViewer)
```

Comment:
>winrt::Windows::UI::Xaml::Controls::ScrollViewer [](start = 10, length = 48)

nit: can we create an alias for this, like winrt::FxScrollViewer or something so we do not have to expand the full namespace everywhere?

Reply:
Done. I called it winrt::FxScrollViewer.